src/components/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+type VisuallyHiddenProps = {
+  children: JSX.Element | JSX.Element[];
+  wrapperAs?: 'span' | 'div';
+};
```

Comment:
I think we can declare the type using the `HTMLAttributes` type from React...

```ts
type VisuallyHiddenProps = (
  | HTMLAttributes<HTMLDivElement>
  | HTMLAttributes<HTMLSpanElement>
) & {
  children: JSX.Element | JSX.Element[] | HTMLElement;
  wrapperAs?: 'span' | 'div';
};
```

What this does, I think, is guard against random props being passed to VisuallyHidden which do not exist on HTMLAttributes.

If I add the above code, and then hover over `...props` in VSCode, I see:

![Screenshot 2021-04-08 at 16 08 35](https://user-images.githubusercontent.com/8282383/114051112-b97cc900-9884-11eb-8abf-208ddb4a69e8.png)

Which helps understand what props can be passed to VisuallyHidden.